### ISSUE TITLE
jupyter console requires 3 carriage returns for each command

### ISSUE THREAD

**Issue by p5a0u9l** [Calysto/matlab_kernel#85]
title checks out

output 
`
In [1]: a=2[MetaKernelApp] ERROR | Exception in message handler:
Traceback (most recent call last):
  File "/Users/paul/.pyenv/versions/matlab/lib/python3.4/site-packages/ipykernel/kernelbase.py", line 228, in displ
    handler(stream, idents, msg)
  File "/Users/paul/.pyenv/versions/matlab/lib/python3.4/site-packages/ipykernel/kernelbase.py", line 541, in is_ct
    reply_content = self.do_is_complete(code)
  File "/Users/paul/.pyenv/versions/matlab/lib/python3.4/site-packages/matlab_kernel/kernel.py", line 165, in do_ie
    Path(tmpdir, "test_complete.m").write_text(code)
AttributeError: 'PosixPath' object has no attribute 'write_text'
/Users/paul/.pyenv/versions/3.4.5/envs/matlab/lib/python3.4/site-packages/jupyter_console/ptshell.py:599: UserWarn.
  warn('The kernel did not respond to an is_complete_request. '
In [1]: a=2

a =

     2

In [2]: b = 2
      :
      :

b =

     2

In [3]:
`

**Comment by blink1073** [comment]
Hmm, that function should exist on `Path`.  What is the output of `python -c "from pathlib import Path; print(Path().write_text)"`?

**Comment by p5a0u9l** [comment]
sorry, I forgot to check this email account. here's the output

`~ » python -c "from pathlib import Path; print(Path().write_text)"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: 'PosixPath' object has no attribute 'write_text'`

**Comment by blink1073** [comment]
Ah, it looks like `write_text` was added in a later version of Python.  Can you please try making this changed to `/Users/paul/.pyenv/versions/matlab/lib/python3.4/site-packages/matlab_kernel/kernel.py` and see if it solves your issue?

```diff
diff --git a/matlab_kernel/kernel.py b/matlab_kernel/kernel.py
index 090b38b..f99bc1f 100644
--- a/matlab_kernel/kernel.py
+++ b/matlab_kernel/kernel.py
@@ -152,7 +152,9 @@ class MatlabKernel(MetaKernel):
         if self.parse_code(code)["magic"]:
             return {"status": "complete"}
         with TemporaryDirectory() as tmpdir:
-            Path(tmpdir, "test_complete.m").write_text(code)
+            path = Path(tmpdir, "test_complete.m")
+            with path.open(mode='w') as f:
+                f.write(code)
             self._matlab.eval(
                 "try, pcode {} -inplace; catch, end".format(tmpdir),
                 nargout=0)
```

**Comment by p5a0u9l** [comment]
that did it. I should have mentioned I'm currently stuck on a version of Matlab that only supports Python 3.4. 

Thank you!

**Comment by blink1073** [comment]
Released as 0.13.1.
